EventSourcing.Core/Aggregates/Aggregate.cs

```diff
@@ -37,15 +37,15 @@ public abstract class Aggregate
   protected Aggregate()
   {
     Id = Guid.NewGuid();
-    Type = GetType().Name;
+    Type = GetType().FullName;
```

Comment:
Doesn't using 'FullName' prevent the dev from freely moving an aggregate to different namespaces when it has an existing history in the EventStore?

But I guess it does not matter since the Type not really used for any rehydration logic.